tox.ini

```diff
@@ -0,0 +1,60 @@
+[tox]
+skipsdist = true
+envlist = py38,py311
```

Comment:
We can. I chose not to because it would add extra build/test time, especially if we test for multiple OSs. But if you think it's the right thing to do, we can list them for sure. Please let me know your preference. If you have none, we will stick with oldest and newest supported.